common/scripts/configs/master_integrations.sql

```diff
@@ -322,6 +322,18 @@ do $$
       values ('577de63321333398d11a1120', 89, 'bitbucketServerBasicAuth', 'Bitbucket Server Basic Auth', 'generic', false, 'generic', 5012, '54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2016-06-01', '2016-06-01');
     end if;
 
+    -- gerritBasicAuth master integration
+    if not exists (select 1 from "masterIntegrations" where "name" = 'gerritBasicAuth' and "typeCode" = 5012) then
+      insert into "masterIntegrations" ("id", "masterIntegrationId", "name", "displayName", "type", "isEnabled", "level", "typeCode", "createdBy", "updatedBy", "createdAt", "updatedAt")
+      values ('577de63321333398d11a1121', 90, 'gerritBasicAuth', 'Gerrit Basic Auth', 'generic', false, 'generic', 5012, '54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2018-08-06', '2018-08-06');
```

Comment:
we already have `bitbucketServerBasic` masterIntegration with id 90. you should change that. Also check for others.